src/main.js

```diff
@@ -37,6 +37,7 @@ require.config({
         // implementations (e.g. cloud-based storage).
         "fileSystemImpl"    : "filesystem/impls/filer/FilerFileSystem",
 
+        "caman"             : "thirdparty/caman/caman.full.min",
```

Comment:
This is probably the right thing to do, but let me play devil's advocate: why not move this into a `<script>` block in `src/htmlContent/image-view.html` so that it only loads if/when we need it vs. always.  It's 76K of code we're loading for everyone, when only some people will use it.